### PR TITLE
Feature #204 - Bierzo wallet. Extension's badge text e2e tests.

### LINKED ISSUE #204: e2e test bierzo-wallet extension's badge icon
This test should create an e2e test checking if the user in bierzo wallet sends a couple of requests for getting identities, the badge text in the extension is updated correctly.

**NOTE**
For implementing this ticket we should assure chrome global is accessible from puppeteer.

### PR DESCRIPTION
**Pull request by albertandrejev**
**Description**
This PR will add E2E tests to Bierzo Wallet and will check extension's badge number incrementation in case of request added to the queue. It will close #204.

### CODE REVIEW

File: packages/bierzo-wallet/package.json [hunk 1]
```diff
@@ -28,7 +30,9 @@
     "format": "prettier --write --loglevel warn './src/**/*.{ts,tsx,json,md,css}'",
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "CI=true react-scripts test '\\.dom\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom"
+    "test-dom": "CI=true react-scripts test '\\.dom\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom",
+    "test-e2e": "CI=true concurrently -s \"first\" -k \"serve -s build -l 3000\" \"react-scripts test '\\.e2e\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom\"",
```
Comment (apanizo) [inline packages/bierzo-wallet/package.json]: What is `concurrently -s \"first\" -k \"serve -s build -l 3000\"`?

File: packages/bierzo-wallet/src/routes/welcome/index.e2e.spec.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    let browser: Browser;
+    let page: Page;
+    let backgroundPage: Page;
+    let extensionPage: Page;
+
+    beforeEach(async (): Promise<void> => {
+      browser = await launchBrowser();
+      page = await createPage(browser);
+      extensionPage = await createExtensionPage(browser);
+      backgroundPage = await getBackgroundPage(browser);
```
Comment (apanizo) [inline packages/bierzo-wallet/src/routes/welcome/index.e2e.spec.ts]: I like the `backgroundPage` approach. The initial approach I had in mind was:
```
const badgeText = await page.evaluate(    <------ Note page here
    (): Promise<string | undefined> => {
        return new Promise(resolve => {
            chrome.browserAction.getBadgeText({}, text => resolve(text));
        });
   },
);
```

I thought having the page tab with the extension installed you already had access to `chrome.browserAction`.

But the `getBackgroundPage` is so cool. It is so damn cool that despite we can have other alternatives we are going to leave it in the repo. 

Just curiosity, did you try my approach?

Comment (albertandrejev) [reply packages/bierzo-wallet/src/routes/welcome/index.e2e.spec.ts]: Yes I tried it, but you do not have access to `chome.browserAction` from it.

File: packages/bierzo-wallet/package.json (hunk 1 above)
Comment (albertandrejev) [reply packages/bierzo-wallet/package.json]: full command is `concurrently -s "first" -k "serve -s build -l 3000" "react-scripts test '\\.e2e\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom"
`concurrently` - run commands in parallel 
`-s "first" -k` - kill all threads after first finished thread and return command execution status code from first finished thread.
`serve -s build -l 3000` - serves web static content from `build` directory on port 3000
`react-scripts test '\\.e2e\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom` - I think you know what it is for.

**Pull request merged**